packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -28,6 +30,20 @@
     }
   }
 }
+
+@mixin card-image {
+  .#{$prefix}--card {
+    &__image-ratio {
+      padding-top: aspect-ratio(4, 3);
+      position: relative;
+    }
+    &__img {
+      position: absolute;
+      top: 0;
+    }
+  }
+}
+
```

Comment:
I'm not sure absolutely positioning the image is the best approach here, should these perhaps be a background image instead?

cc: @annawen1

Reply:
@jeffchew the image is being rendered `Image` component  from Crad pattern, and `_image.scss` has object-fit. If I use `background-image` property here how could I pass image src url to this? help me to understand

Reply:
@annawen1 @andysherman2121 any thoughts on this? I know LeadSpace work is in flight now and the original LeadSpaceCentered used a background image...

Reply:
@jeffchew @mkothur I have a pull request over in carbon that's almost ready to be merged with a couple mixins and bug fixes for their current implementation of aspect ratio.

https://github.com/carbon-design-system/carbon/pull/5255/files#diff-986aa34d579220df16c7b49cfda45a2dR252-R265

Here is the crux of the mixin I submitted, and no position absolute is necessary...

```scss
@mixin ratio () {
 display: flex;

 &::before {
  content: '';
  width: 1px;
  margin-left: -1px;
  float: left;
  height: 0;
 }

 &::after {
  content: '';
  display: table;
  clear: both;
 }
}

.#{$prefix}--card-section {
 &__card {
  .#{$prefix}--image {
   @include ratio;

   &--img {
    height: auto;
    flex: 1;
   }
  }
 }
}
```

> Side note, why is the customClassName for the Image component being applied to children elements?

Reply:
yeah @photodow the image is rendering by Image component inside the Card sub-pattern and from where we are passing image prop to card pattern to handle aspect ratio wrapped that Image component in a div inside of Card pattern and added new scss modifier for that in _card-section.scss.

Reply:
@mkothur the example code I provided shouldn't need an extra `div`, or any absolute positioning.

Reply:
@photodow I have tried with your above snippet the aspect ratio is coming as expected. my be I am using wrong way.